src/workerd/api/streams/internal.c++

```diff
@@ -2551,9 +2551,7 @@ size_t ReadableStreamInternalController::PipeLocked::jsgGetMemorySelfSize() cons
 }
 
 void ReadableStreamInternalController::PipeLocked::jsgGetMemoryInfo(
```

Comment:
If it's empty wouldn't it be simpler to just not implement this function?

Reply:
Was trying to minimize the change set a bit in this so the actual fix is clearer then come back and clean these bits up later, but sure, can do it now